dist/windows/installer-translations/portugueseBR.nsi

```diff
@@ -27,9 +27,9 @@ LangString inst_unist ${LANG_PORTUGUESEBR} "Desinstalando versão anterior."
 ;LangString launch_qbt ${LANG_ENGLISH} "Launch qBittorrent."
 LangString launch_qbt ${LANG_PORTUGUESEBR} "Executar qBittorrent."
 ;LangString inst_requires_64bit ${LANG_ENGLISH} "This installer works only in 64-bit Windows versions."
-LangString inst_requires_64bit ${LANG_PORTUGUESEBR} "This installer works only in 64-bit Windows versions."
+LangString inst_requires_64bit ${LANG_PORTUGUESEBR} "Este instalador somente funciona em versões 64-bit do Windows."
```

Comment:
Again, apologies if this is not the idiomatic way to say it in pt-BR, but what about: "funciona somente" instead of "somente funciona"?

Reply:
Either way works, but since you asked I'll change it. Well, I actually don't know how to edit it after making the pull request...

Reply:
@Unknow0059
Are you using the web editor or plain `git`? using the latter, you need to make a new commit, squash the commits together, and push again. If you are using the web editor, I think you can still make changes, but I'm not sure you can squash commits. If so, I believe someone else can squash them for you, as long as you have enabled "allow edits from maintainers" when submitting this.

Reply:
Web editor. That's enabled, yeah.